sdk/cosmos/azure-cosmos/samples/document_management_async.py

```diff
@@ -546,6 +609,8 @@ async def run_sample():
             await read_items(container)
             await query_items(container, 'SalesOrder1')
             await query_items_with_continuation_token(container)
+            await query_items_with_pagination(container)
+            await query_items_cross_partition_with_pagination(container)
```

Comment:
non-blocking: since we have one that specifically calls out the use of cross_partition queries, should we make the other one partitioned? up to you